app/src/androidTest/java/com/ustwo/boilerplate/base/BaseUiTest.java

```diff
@@ -0,0 +1,18 @@
+package com.ustwo.boilerplate.base;
+
+import android.app.Activity;
+import android.support.annotation.NonNull;
+
+import com.squareup.spoon.Spoon;
+
+public class BaseUiTest {
+
+  protected final void screenshot(@NonNull final Activity activity, @NonNull final String
```

Comment:
Perhaps out of scope here but if the constructor needs to be broken we could break after each param. E.g.

```
protected final void screenshot(@NonNull final Activity activity,
  @NonNull final String screenshotName) {
}
```